### PR TITLE
Email Settings Documentation 

### PR DESCRIPTION
**Pull request by AshtonDavis**
Email settings documentation pulled from 
http://projects.theforeman.org/projects/foreman/wiki/Email_configuration

and formatted in MarkDown.  I put it in Section 3.5 after settings/config.yml

### CODE REVIEW

**Comment by mmoll** [comment]
could you squash the commits?

**Comment by AshtonDavis** [comment]
Commits squashed.  I'll keep note of that for the future, my apologies.

**Comment by stbenjam** [comment]
Thank you @ntent-ashton ! :smile:

**Comment by AshtonDavis** [comment]
Hey, let me know if there are other sections to be pulled over, I'd love to contribute any way I can.

File: _includes/manuals/1.6/3.5.2_configuration_options.md [hunk 1]
```diff
@@ -44,6 +44,67 @@ This boolean options configures whether Foreman will provide support for the Jav
...
+```
+:smtp
+:sendmail
+```
+
+##### Example email.yml Configurations
+
+###### Simple Login Authentication (default settings)
+```YAML
+# Outgoing email settings
```
Comment (stbenjam) [inline _includes/manuals/1.6/3.5.2_configuration_options.md]: @ntent-ashton Can you remove this line? It seems to cause problems, it's being interpretted as a new header:

foo

**Pull request merged**